deploy/helm/sumologic/values.yaml

```diff
@@ -164,6 +164,12 @@ sumologic:
   fluentd:
     ## Option to specify the Fluentd buffer as file/memory.
     buffer: "memory"
+    ## Option to turn autoscaling on for fluentd and specify metrics for HPA.
+    autoscaling:
+      enabled: false
```

Comment:
@vsinghal13 will autoscaling be OFF by default?

Reply:
yes, by default it will be off as of now.